source/docs/magento2/search-engine.md

```diff
@@ -70,3 +70,87 @@ query Search {
 ```
 
 If you are using the default theme or the theme Chocolatine, the search bar should now be visible.
+
+
+## Algolia
+
+<blockquote class="feature--new">
+  _This feature has been added in Front-Commerce `2.13.0`_
+</blockquote>
+
+### Requirements
+
+On Magento's side, you need to [install and configure the Algolia module for Magento2](https://www.algolia.com/doc/integration/magento-2/getting-started/quick-start/?client=php#installation).
+
+In addition, the attribute `category_ids` must be configured as both an indexed attribute and as a facet:
+
+1. under _Stores > Configuration > Algolia Search > Products_, add `category_ids` as _Searchable_ in the _Attributes_ parameter
+1. under _Stores > Configuration > Algolia Search > Instant Search Result Page_, add `category_ids` in the _Facets_ parameter
+
+You can then run the indexer so that the products are indexed in Algolia's index.
+
+### Front-Commerce configuration
+
+First, you need to make sure the Algolia's search client is installed:
+
+```
+npm i algoliasearch
+```
+
+On Front-Commerce side, you need to enable the Algolia datasource by making the changes in your `.front-commerce.js` file similar to:
+
+```diff
+// .front-commerce.js
+-  modules: [],
++  modules: ["./node_modules/front-commerce/modules/datasource-algolia"],
+   serverModules: [
+     { name: "FrontCommerceCore", path: "server/modules/front-commerce-core" },
++    {
++      name: "Magento2Algolia",
++      path: "datasource-algolia/server/modules/magento1-algolia",
```

Comment:
There is a strange feeling here, we add "magento**1**-algolia" as the "Magento**2**Algolia" module

```suggestion
+      path: "datasource-algolia/server/modules/magento2-algolia",
```

Reply:
fixed (integrated into 1f5a6d2)